class Restart(Exception):
    pass

class Exit(Exception):
    pass